UI/ServerConnectWnd.h

```diff
@@ -6,12 +6,19 @@
 #include <GG/ListBox.h>
 
 #include "CUIWnd.h"
-
+#include "../network/Networking.h"
 
 /** server connections window */
 class ServerConnectWnd : public CUIWnd
 {
 public:
+    /** Connection parameters */
+    struct Result {
+        std::string player_name;
+        std::string server_name;
```

Comment:
Is it always a name or sometimes intead an IP address?

Reply:
It's named after variable `server_name` in `HumanClientApp::MultiPlayerGame`. Should them be renamed?